academy-of-management-review.csl

```diff
@@ -82,30 +86,31 @@
   </macro>
   <macro name="access">
     <choose>
-      <if variable="URL">
-        <text variable="URL"/>
-        <group prefix=", ">
-          <date variable="accessed">
-            <date-part name="month" suffix=" "/>
-            <date-part name="day" suffix=", "/>
-            <date-part name="year"/>
-          </date>
-        </group>
+      <if variable="page" match="none">
+        <choose>
+          <if variable="URL">
+            <text variable="URL"/>
+            <choose>
+              <if variable="issued" match="none">
+                <group prefix=", ">
+                  <date variable="accessed">
+                    <date-part name="month" suffix=" "/>
+                    <date-part name="day" suffix=", "/>
+                    <date-part name="year"/>
+                  </date>
+                </group>
```

Comment:
You don't need the `group` here, the `prefix` can go into the `date`, cf. http://citationstyles.org/downloads/specification.html#affixes .